src/controllers/dashboard/plugins/installed.js

```diff
@@ -115,11 +120,13 @@ define(['loading', 'libraryMenu', 'dom', 'globalize', 'cardStyle', 'emby-button'
             });
         }
 
-        menuItems.push({
-            name: globalize.translate('ButtonUninstall'),
-            id: 'delete',
-            icon: 'delete'
-        });
+        if (removable == 'true') {
```

Comment:
```suggestion
        if (removable === 'true') {
```
do we want to do so?